templates/cga-proxy/mainTemplate.json

```diff
@@ -126,7 +126,7 @@
ogICAgZmkKICAgIGlmIFtbICIke1RBR30iID1+IF5oYUVuYWJsZWQgXV07IHRoZW4KICAgICAgICBIQV9FTkFCTEVEPSIke1RBRyMqOn0iCiAgICBmaQpkb25lCgojIFRyeSB0b2tlbiB1bnRpbCBpdCdzIHZhbGlkIG9yIFZNU1MgcmVwbGFjZXMgdGhlIGluc3RhbmNlCiMgTmVlZHMgVk1TUyBpZGVudGl0eSBhZGRlZCB0byB2YXVsdCAobWlnaHQgdGFrZSBhIHdoaWxlKQpsb2dfZW50cnkgIklORk8iICJHZXQgZW5yb2xsbWVudCB0b2tlbiIKZm9yICgoUlVOID0gMTsgOyBSVU4rKykpOyBkbwogICAgQ0dBX1RPS0VOPSIkKGdldF9zZWNyZXQgIiR7QVpVUkVfVE9LRU59IiAiJHtLRVlfVkFVTFRfTkFNRX0iIGNnYS1wcm94eS10b2tlbikiCiAgICBpZiBbWyAiJHtDR0FfVE9LRU46LX0iID1+IF5odHRwW3NdPzovL1teL10rL3Byb3hpZXMvdlswLTldKy9lbnJvbGxtZW50L1swLTlhLWYtXStcP3Byb3h5X2F1dGhfdG9rZW49W15cJl0rXCZ0ZW5hbnRfaWQ9WzAtOWEtZi1dKyQgXV07IHRoZW4KICAgICAgICBsb2dfZW50cnkgIklORk8iICJFbnJvbGxtZW50IHRva2VuIGlzIHZhbGlkIgogICAgICAgIGJyZWFrCiAgICBmaQogICAgbG9nX2VudHJ5ICJJTkZPIiAiUmV0cnlpbmcgZW5yb2xsbWVudCB0b2tlbiByZXF1ZXN0IGZyb20gdmF1bHQgaW4gNSBzZWNvbmRzLiBBdHRlbXB0ICR7UlVOfSIKICAgIHNsZWVwIDUKZG9uZQoKIyBTZXQgcmVkaXMgaW5zdGFuY2Ugd2hlbiBuZWNlc3NhcnkKRVhUUkFfQVJHUz0oKQppZiBbWyAiJHtIQV9FTkFCTEVEOi19IiA9PSAidHJ1ZSIgXV07IHRoZW4KCiAgICBsb2dfZW50cnkgIklORk8iICJIQSBpcyBlbmFibGVkIgoKICAgIGxvZ19lbnRyeSAiSU5GTyIgIkdldCBNaWNyb3NvZnQgcm9vdCBDQXMiCiAgICAjIGh0dHBzOi8vZG9jcy5taWNyb3NvZnQuY29tL2VuLXVzL2F6dXJlL2F6dXJlLWNhY2hlLWZvci1yZWRpcy9jYWNoZS13aGF0cy1uZXcKICAgICMgaHR0cHM6Ly93d3cuZGlnaWNlcnQuY29tL2tiL2RpZ2ljZXJ0LXJvb3QtY2VydGlmaWNhdGVzLmh0bQogICAgY3VybCAtcyBodHRwczovL2NhY2VydHMuZGlnaWNlcnQuY29tL0JhbHRpbW9yZUN5YmVyVHJ1c3RSb290LmNydC5wZW0gPi9ldGMvc3NsL2NlcnRzL2F6dXJlLXJlZGlzLWNhLnBlbQogICAgY3VybCAtcyBodHRwczovL2NhY2VydHMuZGlnaWNlcnQuY29tL0RpZ2lDZXJ0R2xvYmFsUm9vdEcyLmNydC5wZW0gPj4vZXRjL3NzbC9jZXJ0cy9henVyZS1yZWRpcy1jYS5wZW0KICAgICMgaHR0cHM6Ly93d3cubWljcm9zb2Z0LmNvbS9wa2lvcHMvRG9jcy9SZXBvc2l0b3J5Lmh0bQogICAgb3BlbnNzbCB4NTA5IC1pbmZvcm0gREVSIC1pbiBcCiAgICAgICAgPChjdXJsIC1zICJodHRwOi8vd3d3Lm1pY3Jvc29mdC5jb20vcGtpb3BzL2NlcnRzL01pY3Jvc29mdCUyMFJTQSUyMFJvb3QlMjBDZXJ0aWZpY2F0ZSUyMEF1dGhvcml0eSUyMDIwMTcuY3J0IiAtLW91dHB1dCAtKSBcCiAgICAgICAgPj4vZXRjL3NzbC9jZXJ0cy9henVyZS1yZWRpcy1jYS5wZW0KCiAgICBsb2dfZW50cnkgIklORk8iICJHZXQgcmVkaXMgc2VjcmV0cyIKICAgIEVYVFJBX0FSR1MrPSgiLWUiICJSRURJU19IT1NUPSQoZ2V0X3NlY3JldCAiJHtBWlVSRV9UT0tFTn0iICIke0tFWV9WQVVMVF9OQU1FfSIgcmVkaXMtaG9zdCkiKQogICAgRVhUUkFfQVJHUys9KCItZSIgIlJFRElTX1BPUlQ9JChnZXRfc2VjcmV0ICIke0FaVVJFX1RPS0VOfSIgIiR7S0VZX1ZBVUxUX05BTUV9IiByZWRpcy1wb3J0KSIpCiAgICBFWFRSQV9BUkdTKz0oIi1lIiAiUkVESVNfQVVUSD0kKGdldF9zZWNyZXQgIiR7QVpVUkVfVE9LRU59IiAiJHtLRVlfVkFVTFRfTkFNRX0iIHJlZGlzLWF1dGgpIikKICAgIEVYVFJBX0FSR1MrPSgiLWUiICJSRURJU19TU0w9VFJVRSIpCgplbHNlCiAgICBsb2dfZW50cnkgIklORk8iICJIQSBpcyBub3QgZW5hYmxlZCIKZmkKCmxvZ19lbnRyeSAiSU5GTyIgIkluc3RhbGwgcHJveHkiCmN1cmwgLXNMICJodHRwczovL3VybC5hY2Nlc3MuYmFycmFjdWRhLmNvbS9wcm94eS1saW51eCIgfCBiYXNoIC1zIC0tIC11IC1wICI0NDMiIC16IFwKICAgIC10ICIke0NHQV9UT0tFTn0iIFwKICAgICIke0VYVFJBX0FSR1NbQF19IgoKbG9nX2VudHJ5ICJJTkZPIiAiSGFyZGVuIGluc3RhbmNlIgpjdXJsIC1zTCAiaHR0cHM6Ly91cmwuYWNjZXNzLmJhcnJhY3VkYS5jb20vaGFyZGVuLWxpbnV4IiB8IGJhc2ggLXMgLS0KCmxvZ19lbnRyeSAiSU5GTyIgIlJlYm9vdCIKc2h1dGRvd24gLXIgbm93Cg==",
```

Comment:
Confirming if this is intended?

Reply:
Yes, this is the base64 of the bootstrap script for the VM.